client/network.go

```diff
@@ -129,9 +171,22 @@ func (cli *NetworkCli) CmdNetworkInfo(chain string, args ...string) error {
 	if err != nil {
 		return err
 	}
-	if _, err := io.Copy(cli.out, bytes.NewReader(obj)); err != nil {
+
+	networkResource := &networkResource{}
+	if err := json.NewDecoder(bytes.NewReader(obj)).Decode(networkResource); err != nil {
 		return err
 	}
+	fmt.Fprintf(cli.out, "Network Id: %s\n", networkResource.ID)
+	fmt.Fprintf(cli.out, "Network Name: %s\n", networkResource.Name)
+	fmt.Fprintf(cli.out, "Network Type: %s\n", networkResource.Type)
+	if networkResource.Endpoints != nil {
+		for _, endpointResource := range networkResource.Endpoints {
+			fmt.Fprintf(cli.out, "Service Id: %s\n", endpointResource.ID)
+			fmt.Fprintf(cli.out, "Service Name: %s\n", endpointResource.Name)
```

Comment:
Also same comment as before, we dont have to repeat the word "Service" for every field. Rather, we can have a "Service" keyword followed by a bunch of elements such as  "name", "id", etc...